src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import "./Navbar.css";
+import { Link } from "react-router-dom"
+import NavLogo from "../../Assets/nav-logo.png"
+import { useFilter } from '../../Contexts/filter-controller'
+
+const Navbar = () => {
+    const { dispatch, filteredState } = useFilter()
+    return (
+        <nav className="navbar">
+            <div className="nav-logo">
+                <img src={NavLogo} alt="logo" />
+            </div>
+            <div className="nav-items">
+                <Link to="/" className="nav-link sm-title hide-md">About Us</Link>
+                <Link to="/shop" className="nav-link sm-title hide-md">Shop</Link>
+            </div>
+            <div className="nav-space"></div>
+            <div className="nav-icon-items">
+                <div className="nav-search-bar hide-sm">
+                    <input className="search-bar" type="text" placeholder="Search.." />
+                    <i className="fa fa-brands fa-searchengin"></i>
+                </div>
+                <a href="Authentication/login.html">
```

Comment:
You can replace all <a> tags by <Link> tags

Reply:
could you please elaborate. i cant understand what you mean by that.